src/makeCalculator.js

```diff
@@ -37,7 +37,34 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calc = {
+    result: 0,
+
+    add(n) {
+      calc.result += n;
+    },
+    subtract(n) {
+      calc.result -= n;
+    },
+    multiply(n) {
+      calc.result *= n;
+    },
+    divide(n) {
+      calc.result /= n;
+    },
+    reset() {
+      calc.result = 0;
+
+      return calc;
```

Comment:
It's not a mistake, but I think it's not adaptive. For my opinion, more flexible will be if we return `this` instead of name of our object